src/odb/include/odb/db.h

```diff
@@ -9223,6 +9271,116 @@ class dbAccessPoint : public dbObject
   // User Code End dbAccessPoint
 };
 
+class dbPowerDomain : public dbObject
+{
+ public:
+  // User Code Begin dbPowerDomainEnums
+  // User Code End dbPowerDomainEnums
+  const char* getName() const;
+
+  void setPowerSwitch(dbPowerSwitch* power_switch);
```

Comment:
A power domain is not limited to only having one power switch associated with it. It's better to say that there are 0 or more power switches for a given power domain.